.pre-commit-config.yaml

```diff
@@ -125,7 +142,7 @@ repos:
         name: flake8
         additional_dependencies: ["flake8-bugbear==21.9.2"]
   - repo: https://github.com/OCA/pylint-odoo
-    rev: 7.0.2
+    rev: 7.0.5
```

Comment:
The repo https://github.com/OCA/odoo-pre-commit-hooks born with pylint-odoo checks migrated to hooks

But it was done after pyylint_odoo >= 8.0

So, if you add odoo-pre-commit-hooks you will need to update pylint-odoo to >=8.0 to avoid duplicating checks

I meant, pylint-odoo < 8.0 is not compatible with odoo-pre-commit-hooks

cc @sbidoul @pedrobaeza @antonag32